api.go

```diff
@@ -1071,3 +1071,15 @@ func (c *Client) CredContext() *credentials.CredContext {
 		Endpoint: c.endpointURL.String(),
 	}
 }
+
+// GetCreds returns the access creds for the client
+func (c *Client) GetCreds() (string, string, error) {
```

Comment:
Credentials are always 3 entities, you should return the value()

But what is the point of this function?

Reply:
This is not needed with new changes. Will remove.